fdbserver/workloads/ReadWrite.actor.cpp

```diff
@@ -56,7 +56,26 @@ static Future<Version> getInconsistentReadVersion( Database const& db ) {
 	else
 		return lastRV;
 }
-
+ACTOR static Future<Void> slowRandomStorageActor(Database cx) {
```

Comment:
This should slow a random machine rather than a random storage server. That will simplify the code, and in practice most processes in simulation are storage servers.